app-backend/backsplash-server/src/main/resources/application.conf

```diff
@@ -1,14 +1,4 @@
 parallelism {
-
-  dbThreadPoolSize = 20
```

Comment:
I pushed up 81360934c9ab487b8fe9c6973cd560a5aab3eb7a to remove these since it seems like they are no longer in use.

Reply:
Nice, thanks.